src/renderer/store/molecules/Toot.js

```diff
@@ -20,14 +20,17 @@ const Toot = {
           return res.data.reblog
         })
     },
-    unreblog ({ state, commit, rootState }, message) {
+    async unreblog ({ state, commit, rootState }, message) {
```

Comment:
Why you add `async`? I think that it is not required in this function, because there are no `await` keywords.